packages/@webex/internal-plugin-dss/src/config.ts

```diff
@@ -4,6 +4,12 @@
 
 export default {
   dss: {
+    /**
+     * Timeout before DSS request fails, in milliseconds.
+     * @type {Number}
+     */
+    requestTimeout: 1000,
```

Comment:
I don't know much about DSS, maybe it's very quick, but 1s feels like a very short timeout from my experience with Webex backend components

Reply:
@marcin-bazyl based on the metrics the average DSS response time is below 100ms: https://metrics-noram.wbx2.com/goto/5vwQxSCVz I thought 10 times more would be sufficient for timeout

Reply:
Doesn't include network time at any rate, so need to add on maximum expected network delay (which we presumably have no idea), maybe go back to your first number of 6000, since it isn't causing any problems we know of for kms (I know I suggested changing it in the first place)? I really don't know though :/ but would hate to cause this to start failing for no good reason

Reply:
true, the network time was not part of it. KMS use 6000ms as well so it could be a good staring point. We can adjust it later when necessary.